core/pom.xml

```diff
@@ -103,6 +103,12 @@
 			<version>${guava.version}</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+    		<groupId>opennlp</groupId>
+    		<artifactId>maxent</artifactId>
+    		<version>3.0.0</version>
```

Comment:
We fixed a couple of bugs over time, and added new features, usual maintenance (e.g. testing on recent java versions),  so yeah, it probably makes sense to use a recent version when you build something new. Also it supports now multi-threaded training.  

The version you are linking above is 7 years old.